src/PHPMailer.php

```diff
@@ -1571,7 +1571,7 @@ public function preSend()
 
             //Validate From, Sender, and ConfirmReadingTo addresses
             foreach (['From', 'Sender', 'ConfirmReadingTo'] as $address_kind) {
-                $this->{$address_kind} = trim($this->{$address_kind});
+                $this->{$address_kind} = trim($this->{$address_kind} ?? '');
```

Comment:
Unfortunately null coalesce was only introduced in PHP 7.0, and we have BC back to 5.5, so this syntax can't be used. Since these properties are not meant to be nullable, I think a better approach is to stop them being null, like this:

```php
if ($this->{$address_kind} === null) {
    $this->{$address_kind} = '';
}
```
That will solve the trim issue and also fix any unexpected nulls that crop up elsewhere.

Reply:
Yup, thanks for reviewing...
Committed the fix...